### PR TITLE
(MODULES-7109) CHANGELOG edits

### PR DESCRIPTION
**Pull request by clairecadman**
CHANGELOG edits to follow KAC format.

### CODE REVIEW

**Comment by glennsarti** [comment]
I don't think we've come to a consensus on present vs past tense for Changelog.  There was a discussion here (https://github.com/olivierlacan/keep-a-changelog/issues/54) about it.

Typically ChangeLogs are documenting the past so past tense seems to make sense.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,147 +8,150 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
...
 
-- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271))
+- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271)).
 
 ### Fixed
 
-- Upgrade message is improved to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011))
-- Fix timeout handling when the user specified a timeout parameter value of `0` to substitute the default of 300 seconds ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018))
+- Upgraded message to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011)).
+- Timeout handling when the user specifies a timeout parameter value of `0` to substitute the default of 300 seconds ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018)).
```
Comment (jpogran) [inline CHANGELOG.md]: Removing 'fix' from several lines makes it harder to read IMHO

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,147 +8,150 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
...
 ### Added
 
-- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271))
+- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271)).
 
 ### Fixed
 
-- Upgrade message is improved to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011))
-- Fix timeout handling when the user specified a timeout parameter value of `0` to substitute the default of 300 seconds ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018))
+- Upgraded message to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011)).
```
[the remark this answers is not in the document]
Comment (clairecadman) [reply CHANGELOG.md]: Based this on the KAC style, but I see from the tense discussion you don't always follow this — I will change now.

**Comment by RandomNoun7** [comment]
@jpogran Dismissing your review because @clairecadman seems to have addressed your edits.

**Pull request merged**